maintainers/maintainer-list.nix

```diff
@@ -8327,6 +8327,12 @@
     githubId = 4958190;
     name = "Rushmore Mushambi";
   };
+  russell = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add russell``.

Reply:
no problem, split into 2 commits :)